source/elements/oneTBB/source/task_scheduler/task_arena/task_arena_cls.rst

```diff
@@ -343,14 +362,9 @@ to the corresponding NUMA node.
     #include <vector>
 
     int main() {
-        std::vector<oneapi::tbb::numa_node_id> numa_nodes = oneapi::tbb::info::numa_nodes();
-        std::vector<oneapi::tbb::task_arena> arenas(numa_nodes.size());
+        std::vector<oneapi::tbb::task_arena> arenas = oneapi::tbb::create_numa_task_arenas();
         std::vector<oneapi::tbb::task_group> task_groups(numa_nodes.size());
 
```

Comment:
Since `numa_nodes` were removed, make sure these are not used in the remaining code.